app/controllers/idv/phone_controller.rb

```diff
@@ -33,6 +47,20 @@ def new
 
     def create
       result = idv_form.submit(step_params)
+      # Funnel::DocAuth::RegisterStep.new(current_user.id, current_sp&.issuer).
+      #   call(:verify_phone, :update, result.success?)
+      Rails.logger.info(
+        {
+          name: 'event_to_doc_auth_log_token',
+          source: 'proposed',
+          user_id: current_user.id,
+          issuer: current_sp&.issuer,
+          token: :verify_phone,
+          action: :update,
+          success: true,
```

Comment:
The original call has `result.success?` here. Is that important?

When I added these calls to VerifyInfoController, I didn't track success because it wasn't known at the time the call was made (async processing).

Reply:
whoops I should have done that, great catch, thank you

Reply:
```suggestion
          success: result.success?,
```